Webapp/sources/services/hazard_code.py

```diff
@@ -36,17 +36,18 @@
 }
 
 
-def get_multiple_compounds_data(
-    compounds_hazard_codes_list: List[str], physical_forms_list: List[str]
-) -> Tuple[List[int], List[str], List[str], List[str], List[str], List[str], List[str]]:
+# def get_multiple_compounds_data(
```

Comment:
Just delete this instead of commenting it out. Generally you want to avoid commenting out code without an explanation, or it will never get deleted!